src/routes/v2/authenticated/collaborators.ts

```diff
@@ -103,6 +103,23 @@ export class CollaboratorsRouter {
     return res.status(200).json({ role })
   }
 
+  getCollaboratorsStatistics: RequestHandler<
+    { siteName: string },
+    unknown,
+    never,
+    never,
+    { userWithSiteSessionData: UserWithSiteSessionData }
+  > = async (req, res) => {
+    const { siteName } = req.params
+    const statistics = await this.collaboratorsService.getStatistics(siteName)
+
+    // Check for error and throw
+    if (statistics instanceof BaseIsomerError) {
+      throw statistics
+    }
```

Comment:
we shouldn't throw for _expected_ errors

Reply:
Fixed in 4481e22.